### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by matthewdking**
relates #27 

+ create react app
+ schemas on readme #30 
+ pre-commit
+ travis
+ README

### CODE REVIEW

**Comment by matthewdking** [comment]
@finnhodgkin said it was fine to use PostgreSQL and I thought we both thought we hadn't done it for a while so it was better? Can change it though if you aren't happy with it?

**Comment by veredrec** [comment]
No, I'm very happy about it! Just wanted to make sure :+1:

File: README.md [hunk 1]
```diff
@@ -1 +1,89 @@
-# path-18-app
\ No newline at end of file
+# path-18-app
```
Comment (finnhodgkin) [inline README.md]: Path 18

File: README.md [hunk 2]
```diff
@@ -1 +1,89 @@
...
+
+Designed to give young adults in Israel a greater exposure to a wide variety of career paths. We also show users specific Universities they can apply to with their current grades.
+
+### Contributors
+
```
Comment (finnhodgkin) [inline README.md]: I think someone is missing here... :blush:

File: README.md [hunk 3]
```diff
@@ -1 +1,89 @@
...
++ [PostgreSQL](https://www.postgresql.org/)
+
+### Database schemas
+
+**Users**
+
+| user_id | first_name | last_name  | email | grade_bagrut | grade_psychometric | grade_tawjihi |
+| ------- |:----------:|:----------:|:-----:|:------------:|:------------------:|:-------:|
+| 1       |            |            |       |              |                    |         |
+| 2       |            |            |       |              |                    |         |
```
Comment (finnhodgkin) [inline README.md]: I'm not sure about the blank examples here. Just one line with examples or even just the titles without examples...

File: README.md [hunk 4]
```diff
@@ -1 +1,89 @@
...
+| -------- |:----------:|:----------:|:-----:|
+| 1        |            |            |       |
+| 2        |            |            |       |
+| 3        |            |            |       |
+
+**Careers**
+
+| career_id | title   | title_ar| title_he | tagline | tagline_ar | tagline_he | description | description_ar | description_he | image_url | salary_start | salary_ten_year | icon_url |
+| --------- |:-------:|:-------:|:--------:|:-------:|:----------:|:----------:|:-----------:|:-----------:|:-----------:|:---------:|:-----------:|:---------------:|:--------:|
+| 1         | Doctor  | Doctor  | Doctor   |         |            |            |             |             |             |           |             |                 |          |
```
Comment (finnhodgkin) [inline README.md]: Doctor Doctor Doctor. I reckon just scrap the examples.

**Pull request merged**